pages/emailSender.tsx

```diff
@@ -340,12 +351,28 @@ const EmailSender: NextPage = () => {
             <StyledButton
               color="info"
               variant="contained"
-              onClick={() => sendEmails()}
+              onClick={() => handleClickOpen()}
               width="medium"
               disabled={finalMessages.length === 0}
             >
               Send!
             </StyledButton>
+            <Dialog
+              open={open}
+              onClose={handleClose}
+              aria-labelledby="alert-dialog-title"
+              aria-describedby="alert-dialog-description"
+            >
+              <DialogTitle id="alert-dialog-title">
+                {"Are you sure you want to send all emails?"}
```

Comment:
We don't need to add the curly brackets and quotes here - you can just change to Are you sure you want to send all emails?